workers/data_refinery_workers/processors/no_op.py

```diff
@@ -203,6 +203,8 @@ def _convert_affy_genes(job_context: Dict) -> Dict:
     # Related: https://github.com/AlexsLemonade/refinebio/issues/614
     # Related: GSM102671
     is_good_quality = check_output_quality(job_context['output_file_path'])
+    if not is_good_quality:
+        job_context["success"] = True
```

Comment:
If it's not good quality we call it a success? Also it looks like we set `job_context['success']` to true one way or another in a couple lines anyway.